rollup.config.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import cleanup from 'rollup-plugin-cleanup';
+
+const banner = `/*
+ * Copyright ${new Date().getFullYear()} Adobe. All rights reserved.
```

Comment:
```suggestion
 * Copyright 2024 Adobe. All rights reserved.
```

You want to pin the copyright to the oldest possible date, so you establish a longer claim.

Reply:
this takes the current year dynamically when rollup runs. Do you think it is better to hardcode it to 2024?
In any case it is not very relevant, as we are getting rid of the banner for now

Reply:
For templates (like in our ESLint templates), it is a somewhat good idea, as a new file that does not have a copyright date will get the current year. As the file is revised, the year stays the same, so that the older claim stays in tact.